src/main.cpp

```diff
@@ -28,6 +52,22 @@ int main(int argc, char *argv[])
     });
     parser.process(app);
 
+    QSettings settings;
+    QFile file(settings.fileName());
+    qDebug() << "Config file: " << file.fileName();
+    if (!file.exists()) {
+        qDebug() << "WARNING: Settings file " << file.fileName() << "not found!";
+        qDebug() << "Writing new settings file with default values.";
```

Comment:
```suggestion
    if (!file.exists()) {
        qWarning() << "Settings file" << file.fileName() << "not found!";
        qWarning() << "Writing new settings file with default values";
```

Reply:
wouldn't it be nice to inform the user (if they are inspecting the qtphy logs) where the config file is so they can tweak it if they want? Same as with RAUC config?

Reply:
Yeah, I thought about that. As for the RAUC config: There is some additional work to do for it to properly work. Then I'd probably add another command line option, the same way as with the rauc-hawkbit-updater config.

Now, the Qt application level config file is standardized in its path when using the default filename, that is `XDG_CONFIG_HOME="$HOME/.config"`. I'd like to retain this behavior. I am not sure, if we really need to configure the path of this config file to gain any benefit. The RAUC config file is different, as it is an external program that may change in the future or depending on the machine.

The printing may be reduced to a `qInfo()` instead of `qWarning()`, as it is not really a problem if the config file does not exist. The config file path is also not something the user should care about. If we allow for configuration options, they either should be available via command line options or be directly available in the qtphy app to interact with. The config file is not meant to be edited directly, as this may make it invalid.

Reply:
Ok if the config file is not meant to be edited directly (although it is a nice way of adding pages during development/debugging), then I agree, there's no need to print it out :).

Reply:
I admit, our use case is a little special, as we want to provide specific config files disabling certain pages. So yeah, only meant for development/debugging/experienced users. :sweat_smile: